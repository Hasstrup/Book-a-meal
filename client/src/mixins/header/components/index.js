import React from 'react';
import { LogOutUser } from '../../../actions/users/'

/**
   * @name LoggedOutHeader
   * @description This header would only be rendered when there is no user in session
   * @param {object} props the props containing the history object for navigation
   * @returns {function} React Component
   */
export const LoggedOutHeader = ({ history }) => (
  <header>
    <div className="header">
      <div className="home-tag">
        <p className="logo" onClick={() => { history.push('/'); }}> Book <br />
            a meal
        </p>
      </div>
      <div className="header-buttons">
        <p className="h1-login-button"> Login </p>
        <p> Sign Up</p>
        <p className="h1-login-button" onClick={() => { history.push('/catalogue'); }}> Catalogue </p>
      </div>
    </div>
  </header>
);


  /**
   * @name LoggedInHeader
   * @description This header would only be rendered when a user is logged in
   * @param {object} props the props containing the history object for navigation
   * @returns {function} React Component
   */
export const LoggedInHeader = ({ history }) => (
  <header>
    <div className="header">
      <div className="home-tag">
        <p className="logo" onClick={() => { history.push('/'); }}> Book <br />
            a meal
          </p>
      </div>
      <div className="header-buttons">
        <p className="h1-login-button" onClick={() => { history.push('/orders')}}> Cart & Orders </p>
        <p onClick={() => { history.push('/profile')}} > Workstation </p>
        <p className="h1-login-button" onClick={() => { history.push('/catalogue')}}> Catalogue </p>
        <p className="h1-login-button" onClick={() => { LogOutUser(history) }}> Logout </p>
      </div>
    </div>
  </header>
);
