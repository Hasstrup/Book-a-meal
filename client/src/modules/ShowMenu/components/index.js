import ShowMeal from './ShowMeal';
import ShowMenuDetails from './ShowMenuDetails';
import VendorDetails from './VendorDetails';

export const ComponentList = [ShowMeal, ShowMenuDetails, VendorDetails];
export const PropList = [{}, { meals: [] }, { menu: { meals: [], description: '', name: '', kitchen: { image: '', name: '', description: '' }}}];
export const NameList = ['Show Meal', 'ShowMenuDetails', 'VendorDetails'];

