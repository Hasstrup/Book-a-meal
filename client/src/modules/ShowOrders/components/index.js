import CurrentOrder from './CurrentOrder';
import OrderHistoryGrid from './OrderHistoryGrid';
import { OrderHistoryComponent as RenderMainHistory } from './RenderMainHistory';

export const ComponentList = [
  CurrentOrder,
  OrderHistoryGrid,
  RenderMainHistory
];

export const PropList = [
  { meals: [], history: () => {} },
  { orders: [], title: '' },
  { orders: [] }
];

export const NameList = [
  'Current Order',
  'OrderHistoryGrid',
  'RenderMainHistory'
];
