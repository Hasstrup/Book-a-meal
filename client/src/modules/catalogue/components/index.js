import { CatalogueFirstRow, CatalogueGridMainX } from '../index';

export const ComponentList = [CatalogueFirstRow, CatalogueGridMainX];
export const PropList = [{}, { menus: [] }];
export const NameList = ['Catalogue Preamble', 'Catalogue Grid'];

