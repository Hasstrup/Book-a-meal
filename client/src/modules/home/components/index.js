import AuthComponent from './auth';
import BannerComponent from './banner';
import FloatingComponent from './floater';

export const ComponentList = [AuthComponent, BannerComponent, FloatingComponent];
export const PropList = [{ logInUser: () => {}, createUser: () => {} }, {}, {}];
export const NameList = ['Auth Component', 'Banner Component', 'Floating Component'];