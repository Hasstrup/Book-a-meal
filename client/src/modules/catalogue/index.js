import React, { Component } from 'react';
import { connect } from 'react-redux';
import PropTypes from 'prop-types';
import '../styles/catalogue.scss';
import { MainCatalogue } from '../../mixins/cards/singleMenu';
import { FetchCatalogue } from '../../actions/menus/';


class Catalogue extends Component {
  static propTypes = {
    dispatch: PropTypes.func.isRequired,
    menus: PropTypes.arrayOf(PropTypes.object).isRequired,
    history: PropTypes.object.isRequired
  }

  componentDidMount = () => {
    if (!this.props.menus) return this.fetchMenus();
  }

  fetchMenus = () => this.props.dispatch(FetchCatalogue(this.props.history));

  render = () => (
    <div className="main-body">
      <CatalogueFirstRow />
      <Selectors />
      <CatalogueGridMain history={this.props.history} menus={this.props.menus || []} />
    </div>
  );
}

/*= ========== Components ============================ */

const CatalogueFirstRow = () => (
  <div className="introductory-grid">
    <div className="side-one-IT">
      <div className="side-one-child-1">
        <p className="side-one-IT-cat"> Catalogue </p>
        <div className="side-one-trending"> Today</div>
      </div>
      <p className="side-one-IT-option"> <span> 237 </span> Menus </p>
      <p className="side-one-IT-option"> <span> 1034</span> Meal options </p>
      <p className="side-one-IT-option"> <span> 101 </span> Vendors</p>
    </div>

    <div className="side-two-IT">
      <p className="side-two-IT-info">
                Sharing a menu is super easy! There's no restrictions
                whatsoever, all you have to do set up your kitchen/catering
                service, add some meal options, set up a menu and we're good
                to go .
      </p>
      <p className="side-two-IT-info" id="side-two-share-button"> Share your menu now</p>
    </div>
  </div>
);


const Selectors = () => (
  <div className="options-grid">
    <p> Top </p>
    <p> Featured </p>
    <p> All </p>
    <p> Favourites </p>
    <p> Promote </p>
  </div>
);


const CatalogueGridMain = props => (
  <div className="main-items-grid">
    { /* remember to get some mock || main data to fill in this place */ }
    { MainCatalogue(props.menus)(props) }
  </div>
);

const mapStateToProps = state => ({
  menus: state.menus.catalog
});
// these should be refactored after the application is completed right
export default connect(mapStateToProps)(Catalogue);

