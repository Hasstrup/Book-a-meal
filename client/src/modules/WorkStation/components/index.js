import RenderMeals from './meals';
import RenderOrdersComponent from './orders';
import UserBio from './UserBio';
import { WorkStation as WorkStationMain } from './workstation-main';
import { MenuOfTheDayContainer as MenuOfTheDay } from './workstation-main/modules/MenuOfTheDay';


export const ComponentList =
[
  RenderMeals,
  RenderOrdersComponent,
  UserBio,
  WorkStationMain,
  MenuOfTheDay
];

export const PropList = [
  { handleSubmit: () => {}, meals: [] },
  {},
  {
    user: { firstname: '', email: '', bio: '' },
    handleChange: () => {},
    handleSubmit: () => {}
  },
  {
    kitchen: { name: '', description: '', bio: '' },
    meals: [],
    handleSubmit: () => {}
  },
  {
    dispatch: () => {},
    ofTheDay: { name: '', meals: [], description: '' },
    meals: []
  }

];
export const NameList = [
  'RenderMeals',
  'RenderOrdersComponent',
  'User Bio',
  'WorkStation Main',
  'MenuOfTheDayContainer'
];

