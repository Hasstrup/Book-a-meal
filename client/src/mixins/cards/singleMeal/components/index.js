import React from 'react';
import { changePhoto, HideMealCard } from '../../../../modules/WorkStation/utils';
import { AddToCart } from '../../../../actions/orders/';


const defaultUrl = 'https://images.unsplash.com/photo-1453831362806-3d5577f014a4?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=9dd8da96be0724ab84e4147d428f6bba&auto=format&fit=crop&w=500&q=60';
/**
 * @name SingleMealCard
 * @returns {function} React Component
 * @param {Object} Props The props passed to the item
 * wshould contain the current kitchen the item and the item;
 * @description Single Meal Card component, recieves data from it's parent container
 */
const SingleMealCard = ({ meal, kitchen, handleDelete, handleEdit, displayText, wantsToEdit, renderEditable, dispatch }) => (
    <div className="display-menu-item" id={meal.id} onMouseLeave={() => { HideMealCard(meal.id) }}>
      <div className="order-add-item">
        <img src={meal.image || defaultUrl} alt={meal.name} id={`image-${meal.id}`} className="main-meal-item-img" />
        <div className={`change-preview-image modifiable${meal.id}`} style={{ opacity: wantsToEdit ? 1 : 0, zIndex: wantsToEdit ? 5 : -1, display: wantsToEdit ? 'block' : 'none' }}>
          <p className="label-for-input"> Click to change Image </p>
          <input type="file" className={`input-file image-${meal.id}`} name="input-file" onChange={() => { changePhoto(`image-${meal.id}`)}} accept=".jpg, .png, .jpeg" />
        </div>
      </div>
      <div className="item-details">
        <div className="item-title-and-description">
          <p className={`item-title-main editable editable${meal.id}`}> {meal.name || ''} </p>
          <p className={`item-description editable editable${meal.id}`}> {meal.description || ''} </p>
        </div>
        <div className="buttons-array-and-togglers">
          {
          kitchen && kitchen.id === meal.kitchenId ?
          <p className="option-options"> { renderEditable && <span className="edit-meal-option" onClick={handleEdit} id={`target${meal.id}`}> {displayText} </span> } { wantsToEdit ? null : <span className="delete-meal-option" onClick={handleDelete}> Delete </span> } </p>
          : <p className="option-options" onClick={() => dispatch(AddToCart()(meal))}> Order </p>
        }
  
          <p className="option-options2" > N{meal.price} </p>
        </div>
      </div>
    </div>
);

export default SingleMealCard;