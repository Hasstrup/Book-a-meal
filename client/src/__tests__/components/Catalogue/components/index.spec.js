import TestComponent from '../../../TestComponents';
import { ComponentList, PropList, NameList } from '../../../../modules/catalogue/components';

(async () => {
  await TestComponent(ComponentList, NameList, PropList, 'Catalogue Components');
})();