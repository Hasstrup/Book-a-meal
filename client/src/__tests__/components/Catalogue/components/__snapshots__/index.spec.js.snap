// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Catalogue Components Catalogue Grid should render correctly on shallow render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CatalogueGridMainX
    menus={Array []}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [],
      "className": "main-items-grid",
    },
    "ref": null,
    "rendered": Array [],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [],
        "className": "main-items-grid",
      },
      "ref": null,
      "rendered": Array [],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`Catalogue Components Catalogue Preamble should render correctly on shallow render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CatalogueFirstRow />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="side-one-IT"
        >
          <div
            className="side-one-child-1"
          >
            <p
              className="side-one-IT-cat"
            >
               Catalogue 
            </p>
            <div
              className="side-one-trending"
            >
               Today
            </div>
          </div>
          <p
            className="side-one-IT-option"
          >
             
            <span>
               237 
            </span>
             Menus 
          </p>
          <p
            className="side-one-IT-option"
          >
             
            <span>
               1034
            </span>
             Meal options 
          </p>
          <p
            className="side-one-IT-option"
          >
             
            <span>
               101 
            </span>
             Vendors
          </p>
        </div>,
        <div
          className="side-two-IT"
        >
          <p
            className="side-two-IT-info"
          >
            Sharing a menu is super easy! There's no restrictions whatsoever, all you have to do set up your kitchen/catering service, add some meal options, set up a menu and we're good to go .
          </p>
          <p
            className="side-two-IT-info"
            id="side-two-share-button"
          >
             Share your menu now
          </p>
        </div>,
      ],
      "className": "introductory-grid",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="side-one-child-1"
            >
              <p
                className="side-one-IT-cat"
              >
                 Catalogue 
              </p>
              <div
                className="side-one-trending"
              >
                 Today
              </div>
            </div>,
            <p
              className="side-one-IT-option"
            >
               
              <span>
                 237 
              </span>
               Menus 
            </p>,
            <p
              className="side-one-IT-option"
            >
               
              <span>
                 1034
              </span>
               Meal options 
            </p>,
            <p
              className="side-one-IT-option"
            >
               
              <span>
                 101 
              </span>
               Vendors
            </p>,
          ],
          "className": "side-one-IT",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <p
                  className="side-one-IT-cat"
                >
                   Catalogue 
                </p>,
                <div
                  className="side-one-trending"
                >
                   Today
                </div>,
              ],
              "className": "side-one-child-1",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": " Catalogue ",
                  "className": "side-one-IT-cat",
                },
                "ref": null,
                "rendered": " Catalogue ",
                "type": "p",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": " Today",
                  "className": "side-one-trending",
                },
                "ref": null,
                "rendered": " Today",
                "type": "div",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                " ",
                <span>
                   237 
                </span>,
                " Menus ",
              ],
              "className": "side-one-IT-option",
            },
            "ref": null,
            "rendered": Array [
              " ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": " 237 ",
                },
                "ref": null,
                "rendered": " 237 ",
                "type": "span",
              },
              " Menus ",
            ],
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                " ",
                <span>
                   1034
                </span>,
                " Meal options ",
              ],
              "className": "side-one-IT-option",
            },
            "ref": null,
            "rendered": Array [
              " ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": " 1034",
                },
                "ref": null,
                "rendered": " 1034",
                "type": "span",
              },
              " Meal options ",
            ],
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                " ",
                <span>
                   101 
                </span>,
                " Vendors",
              ],
              "className": "side-one-IT-option",
            },
            "ref": null,
            "rendered": Array [
              " ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": " 101 ",
                },
                "ref": null,
                "rendered": " 101 ",
                "type": "span",
              },
              " Vendors",
            ],
            "type": "p",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <p
              className="side-two-IT-info"
            >
              Sharing a menu is super easy! There's no restrictions whatsoever, all you have to do set up your kitchen/catering service, add some meal options, set up a menu and we're good to go .
            </p>,
            <p
              className="side-two-IT-info"
              id="side-two-share-button"
            >
               Share your menu now
            </p>,
          ],
          "className": "side-two-IT",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Sharing a menu is super easy! There's no restrictions whatsoever, all you have to do set up your kitchen/catering service, add some meal options, set up a menu and we're good to go .",
              "className": "side-two-IT-info",
            },
            "ref": null,
            "rendered": "Sharing a menu is super easy! There's no restrictions whatsoever, all you have to do set up your kitchen/catering service, add some meal options, set up a menu and we're good to go .",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": " Share your menu now",
              "className": "side-two-IT-info",
              "id": "side-two-share-button",
            },
            "ref": null,
            "rendered": " Share your menu now",
            "type": "p",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="side-one-IT"
          >
            <div
              className="side-one-child-1"
            >
              <p
                className="side-one-IT-cat"
              >
                 Catalogue 
              </p>
              <div
                className="side-one-trending"
              >
                 Today
              </div>
            </div>
            <p
              className="side-one-IT-option"
            >
               
              <span>
                 237 
              </span>
               Menus 
            </p>
            <p
              className="side-one-IT-option"
            >
               
              <span>
                 1034
              </span>
               Meal options 
            </p>
            <p
              className="side-one-IT-option"
            >
               
              <span>
                 101 
              </span>
               Vendors
            </p>
          </div>,
          <div
            className="side-two-IT"
          >
            <p
              className="side-two-IT-info"
            >
              Sharing a menu is super easy! There's no restrictions whatsoever, all you have to do set up your kitchen/catering service, add some meal options, set up a menu and we're good to go .
            </p>
            <p
              className="side-two-IT-info"
              id="side-two-share-button"
            >
               Share your menu now
            </p>
          </div>,
        ],
        "className": "introductory-grid",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="side-one-child-1"
              >
                <p
                  className="side-one-IT-cat"
                >
                   Catalogue 
                </p>
                <div
                  className="side-one-trending"
                >
                   Today
                </div>
              </div>,
              <p
                className="side-one-IT-option"
              >
                 
                <span>
                   237 
                </span>
                 Menus 
              </p>,
              <p
                className="side-one-IT-option"
              >
                 
                <span>
                   1034
                </span>
                 Meal options 
              </p>,
              <p
                className="side-one-IT-option"
              >
                 
                <span>
                   101 
                </span>
                 Vendors
              </p>,
            ],
            "className": "side-one-IT",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <p
                    className="side-one-IT-cat"
                  >
                     Catalogue 
                  </p>,
                  <div
                    className="side-one-trending"
                  >
                     Today
                  </div>,
                ],
                "className": "side-one-child-1",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": " Catalogue ",
                    "className": "side-one-IT-cat",
                  },
                  "ref": null,
                  "rendered": " Catalogue ",
                  "type": "p",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": " Today",
                    "className": "side-one-trending",
                  },
                  "ref": null,
                  "rendered": " Today",
                  "type": "div",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  " ",
                  <span>
                     237 
                  </span>,
                  " Menus ",
                ],
                "className": "side-one-IT-option",
              },
              "ref": null,
              "rendered": Array [
                " ",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": " 237 ",
                  },
                  "ref": null,
                  "rendered": " 237 ",
                  "type": "span",
                },
                " Menus ",
              ],
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  " ",
                  <span>
                     1034
                  </span>,
                  " Meal options ",
                ],
                "className": "side-one-IT-option",
              },
              "ref": null,
              "rendered": Array [
                " ",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": " 1034",
                  },
                  "ref": null,
                  "rendered": " 1034",
                  "type": "span",
                },
                " Meal options ",
              ],
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  " ",
                  <span>
                     101 
                  </span>,
                  " Vendors",
                ],
                "className": "side-one-IT-option",
              },
              "ref": null,
              "rendered": Array [
                " ",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": " 101 ",
                  },
                  "ref": null,
                  "rendered": " 101 ",
                  "type": "span",
                },
                " Vendors",
              ],
              "type": "p",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <p
                className="side-two-IT-info"
              >
                Sharing a menu is super easy! There's no restrictions whatsoever, all you have to do set up your kitchen/catering service, add some meal options, set up a menu and we're good to go .
              </p>,
              <p
                className="side-two-IT-info"
                id="side-two-share-button"
              >
                 Share your menu now
              </p>,
            ],
            "className": "side-two-IT",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Sharing a menu is super easy! There's no restrictions whatsoever, all you have to do set up your kitchen/catering service, add some meal options, set up a menu and we're good to go .",
                "className": "side-two-IT-info",
              },
              "ref": null,
              "rendered": "Sharing a menu is super easy! There's no restrictions whatsoever, all you have to do set up your kitchen/catering service, add some meal options, set up a menu and we're good to go .",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": " Share your menu now",
                "className": "side-two-IT-info",
                "id": "side-two-share-button",
              },
              "ref": null,
              "rendered": " Share your menu now",
              "type": "p",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
